admin/spam/views.py

```diff
@@ -1,9 +1,37 @@
-from django.http import HttpResponse
+from django.shortcuts import render, render_to_response
+from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
+
+from .serializers import serialize_comments, retrieve_comment
```

Comment:
Just like we talked with @leb2dg about there are DRF serializers for comments that you could import and use/subclass.